bridge-cli/src/omni_connector_command.rs

```diff
@@ -43,15 +61,40 @@ pub enum OmniConnectorSubCommand {
         #[command(flatten)]
         config_cli: CliConfig,
     },
-    EvmFinTransfer {
+    NearFinTransfer {
+        #[clap(short, long)]
+        token_id: String,
+        #[clap(short, long)]
+        account_id: String,
+        #[clap(short, long)]
+        storage_deposit_amount: Option<u128>,
+        #[clap(short, long)]
+        source_chain_id: u8,
+        #[clap(short, long)]
+        vaa: String,
+        #[command(flatten)]
+        config_cli: CliConfig,
+    },
+
+    EvmDeployToken {
+        #[clap(short, long)]
+        source_chain_id: u8,
```

Comment:
Arguments are used during the cli input. Example:
```bash
cr -- testnet omni-connector bind-token-wormhole --source-chain-id '<source-chain-id>' --vaa '<hex-vaa>' --config-file config.json
```

Source chain id can be found [here](https://github.com/Near-One/omni-bridge/blob/51471678cadcfe38d73a88289a8f82dad535f416/near/omni-types/src/lib.rs#L159-L171):

```txt
0 => Eth
1 => Near
2 => Sol
3 => Arb
4 => Base
```

I don't really understand what do you mean by combining the code. I'd rather finish working on cli later, it's not a highest priority right now. I just made sure to refactor code, I wasn't improving it on the cli's part

Reply:
I mean that we need to add help for a command like this:
```
cargo run -- testnet omni-connector wormhole-bind-token --help
```
To explain meaning of each arg. 

This should be really simple to do—probably just adding a description or something. I’m sure it can be easily googled.

Reply:
https://github.com/Near-One/bridge-sdk-rs/issues/58